### PR TITLE
Add notifications section to user settings, including Direct Message setting

### PR DESCRIPTION
**Pull request by robbiecarlton**
Depends on this https://github.com/Hylozoic/hylo-node/pull/206

### CODE REVIEW

File: src/containers/user/UserSettings.js
Comment (levity) [inline src/containers/user/UserSettings.js:416]: Let's link to the iOS and Android apps here, e.g.:

```
Download our <a href="...">iOS</a> or <a href="...">Android</a> apps to receive push notifications.
```

File: src/containers/user/UserSettings.js
Comment (levity) [inline src/containers/user/UserSettings.js]: i'd much rather you used `sendGraphqlQuery` as-is rather than this. if you don't like having to specify a subject and id that go unused, the thing to do might be to move them into the options arg and just have the first arg be `query`.

File: src/actions/graphql.js
Comment (levity) [inline src/actions/graphql.js]: please change my use of `sendGraphqlQuery` in People.js to work with this change

File: src/containers/user/UserSettings.js
Comment (levity) [inline src/containers/user/UserSettings.js]: maybe do this just-in-time, i.e. when the section is opened?

**Comment by robbiecarlton** [comment]
I've made those changes, good to merge now? Once I get the thumbs up I'll merge the relevant parts into https://github.com/Hylozoic/hylo-redux/pull/208

**Review by levity — APPROVED** [review]

**Pull request merged**
